types/types.go

```diff
@@ -356,12 +356,12 @@ type ThrottleDevice struct {
 // ShellCommand is a string or list of string args.
 //
 // When marshaled to YAML, nil command fields will be omitted if `omitempty`
-// is specified as a struct tag. Explicitly empty commands (i.e. `[]` or `''`)
+// is specified as a struct tag. Explicitly empty commands (i.e. `[]` or `”`)
```

Comment:
Is this change intentional?

Reply:
don't know why, I get both Goland and vscode to make this change on save :'(